<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1114</width>
    <height>570</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="1" column="0">
     <widget class="QGroupBox" name="groupBox">
      <property name="title">
       <string>Display Level :</string>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <widget class="QRadioButton" name="pFatalBtn">
         <property name="text">
          <string>Fatal</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QRadioButton" name="pErrorBtn">
         <property name="text">
          <string>Error</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QRadioButton" name="pWarnBtn">
         <property name="text">
          <string>Warn</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QRadioButton" name="pInfoBtn">
         <property name="text">
          <string>I&amp;nfo</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QRadioButton" name="pDebugBtn">
         <property name="text">
          <string>Debug</string>
         </property>
         <property name="checked">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QRadioButton" name="pTraceBtn">
         <property name="text">
          <string>Trace</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item row="1" column="1">
     <widget class="QPlainTextEdit" name="pErrorsDisplay">
      <property name="plainText">
       <string/>
      </property>
     </widget>
    </item>
    <item row="1" column="2">
     <widget class="QGroupBox" name="groupBox_2">
      <property name="title">
       <string>Log Level :</string>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <widget class="QRadioButton" name="pLogFatalBtn">
         <property name="text">
          <string>Fatal</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QRadioButton" name="pLogErrorBtn">
         <property name="text">
          <string>Error</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QRadioButton" name="pLogWarnBtn">
         <property name="text">
          <string>Warn</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QRadioButton" name="pLogInfoBtn">
         <property name="text">
          <string>Info</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QRadioButton" name="pLogDebugBtn">
         <property name="text">
          <string>Debug</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QRadioButton" name="pLogTraceBtn">
         <property name="text">
          <string>Trace</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item row="2" column="0" colspan="3">
     <widget class="QPushButton" name="pSendBtn">
      <property name="text">
       <string>Send</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1114</width>
     <height>28</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="pStatusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
